src/audio/eq_fir/eq_fir.c

```diff
@@ -530,49 +446,37 @@ static int eq_fir_copy(struct comp_dev *dev)
 	 * break the delay line alignment if called with odd number of frames
 	 * so it can't be used here.
 	 */
-	if (cl.frames >= 2) {
-		n = (cl.frames >> 1) << 1;
 
-		/* Run EQ function */
-		eq_fir_process(dev, source_c, sink_c, n,
-			       n * cl.source_frame_bytes,
-			       n * cl.sink_frame_bytes);
-	}
+	frame = input_buffers->size;
```

Comment:
`frame_count `would be more descriptive?

Reply:
sure. Thanks!